src/haddock/modules/analysis/rmsdmatrix/defaults.yaml

```diff
@@ -0,0 +1,22 @@
+max_models:
+  default: 10000
+  type: integer
+  min: 1
+  max: 20000
+  title: Maximum number of models to calculate RMSD matrix
+  short: No short description yet
+  long: No long description yet
+  group: ''
+  explevel: easy
+resdic_:
+  default: []
+  type: list
+  minitems: 0
+  maxitems: 100
+  title: No title yet
```

Comment:
Please add a description in the `title`, `short`, and `long` of the parameters in boty `defaults.yaml`. You can copy those from the docstrings you wrote if you find appropriate.